Jellyfin.Api/Controllers/PlaylistsController.cs

```diff
@@ -92,12 +92,136 @@ public class PlaylistsController : BaseJellyfinApiController
             Name = name ?? createPlaylistRequest?.Name,
             ItemIdList = ids,
             UserId = userId.Value,
-            MediaType = mediaType ?? createPlaylistRequest?.MediaType
+            MediaType = mediaType ?? createPlaylistRequest?.MediaType,
+            Shares = createPlaylistRequest?.Shares.ToArray(),
+            OpenAccess = createPlaylistRequest?.OpenAccess
         }).ConfigureAwait(false);
 
         return result;
     }
 
+    /// <summary>
+    /// Get a playlist's shares.
+    /// </summary>
+    /// <param name="playlistId">The playlist id.</param>
+    /// <returns>
+    /// A list of <see cref="Share"/> objects.
+    /// </returns>
+    [HttpGet("{playlistId}/Shares")]
```

Comment:
```suggestion
    [HttpGet("{playlistId}/User")]
```

Makes more sense because I suggested that path for the add/remove operations